draft-ietf-jsonpath-base.md

```diff
@@ -1104,6 +1094,57 @@ Some examples:
 | `[3,4,5]` | `$[?index(@)==2]`<br>`$[?index(@)==17]` | `[5]`<br>`[]` | Select array element |
 | `{"a":{"b":{5},c:0}}` | `$[?@.b==5 && !@.c]` | `[{"b":{5},c:0}]` | Existence  |
 
+
+## Normalized Paths
+
+A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node. For example,
+the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
+`$['book'][3]` and `$['book'][5]`. For a given JSON document, there is a one to one correspondence between the document's
+nodes and the Normalized Paths that identify these nodes.
+
+An example application of Normalized Paths is a JSONPath implementation which outputs Normalized Paths
+instead of, or in addition to, the values identified by these paths.
```

Comment:
A JSONPath implementation may output Normalized Paths
instead of, or in addition to, the values identified by these paths

Reply:
Adopted. Thanks.